spaces/y3d/index.json

```
@@ -0,0 +1,17 @@
+{
+  "key": "Y3D",
```

The key of your space must also be the folder name and skin class name.
```suggestion
  "key": "y3d",
```

---

I am too careless